src/components/Heroes/Ranking.jsx

```diff
@@ -47,7 +39,7 @@ class Ranking extends Component {
     return (
       <div>
         {isLoading || isError || rankings === null ?
-          renderLoading() : renderRanking(hero, rankings)}
+          <Spinner /> : renderRanking(hero, rankings)}
```

Comment:
Should we show a spinner if there is an error or rankings === null? Should we instead show some sort of error message or nothing?